packages/@aws-cdk/aws-logs-destinations/lib/kinesis.ts

```diff
@@ -0,0 +1,23 @@
+import iam = require('@aws-cdk/aws-iam');
+import kinesis = require('@aws-cdk/aws-kinesis');
+import logs = require('@aws-cdk/aws-logs');
+import { Construct } from '@aws-cdk/cdk';
+
+/**
+ * Use a Kinesis stream as the destination for a log subscription
+ */
+export class KinesisDestination implements logs.ILogSubscriptionDestination {
+  constructor(private readonly stream: kinesis.IStream) {
+  }
+
+  public bind(scope: Construct, _sourceLogGroup: logs.ILogGroup): logs.LogSubscriptionDestination {
+    // Following example from https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/SubscriptionFilters.html#DestinationKinesisExample
+    // Create a role to be assumed by CWL that can write to this stream and pass itself.
```

Comment:
What happens if we want to bind two Kinesis streams? We need only a single role,
Right?

Reply:
A single role with multiple statements, yes. The role belongs conceptually to the log group, I would say.

Aren't we better off return an array of policystatements to be added to the role, instead of the role itself?